.env

```diff
@@ -0,0 +1 @@
+REACT_APP_CONTENTFUL_ENDPOINT=https://preview.contentful.com/spaces/27o4msepdle6/entries?access_token=DpOAFJ5_WtZofKkEmrTWKdyDGW5DP0aIESAl17Adysg
```

Comment:
see: https://www.contentfulcommunity.com/t/should-i-keep-access-tokens-secret/457/2
about commiting this access_token
> ...that being said, allowing the public to learn your space Id and Content Delivery API (CDA) token isn’t a security issue if that space holds only content meant for public consumption. The reasons are:

> CDA is read-only. No one can use its token to change your content, so that’s safe.
Content is public in this scenario, so the worst that can happen is someone very technical firing up a REST client and getting it without going through your site - which isn’t very legible, as it’s all JSON payloads